### PR TITLE
UICONSET-162: Save, edit Authorization policies for consortium

### PR DESCRIPTION
**Pull request by alisher-epam**
## Purpose

[UICONSET-162](https://folio-org.atlassian.net/browse/UICONSET-162): Save, edit Authorization policies for consortium.
There is related PR - https://github.com/folio-org/stripes-authorization-components/pull/29
## Approach

https://github.com/user-attachments/assets/63097035-0a37-473b-8424-614e242411fc

## Pre-Merge Checklist
Before merging this PR, please go through the following list and take appropriate actions.
- [ ] I've added appropriate record to the CHANGELOG.md
- Does this PR meet or exceed the expected quality standards?
  - [ ] Code coverage on new code is 80% or greater
  - [ ] Duplications on new code is 3% or less
  - [ ] There are no major code smells or security issues
- Does this introduce breaking changes?
  - [ ] If any API-related changes - okapi interfaces and permissions are reviewed/changed correspondingly
  - [ ] There are no breaking changes in this PR.

If there are breaking changes, please **STOP** and consider the following:

- What other modules will these changes impact?
- Do JIRAs exist to update the impacted modules?
  - [ ] If not, please create them
  - [ ] Do they contain the appropriate level of detail?  Which endpoints/schemas changed, etc.
  - [ ] Do they have all they appropriate links to blocked/related issues?
- Are the JIRAs under active development?
  - [ ] If not, contact the project's PO and make sure they're aware of the urgency.
- Do PRs exist for these changes?
  - [ ] If so, have they been approved?

Ideally all of the PRs involved in breaking changes would be merged in the same day to avoid breaking the folio-testing environment.  Communication is paramount if that is to be achieved, especially as the number of intermodule and inter-team dependencies increase.

While it's helpful for reviewers to help identify potential problems, ensuring that it's safe to merge is ultimately the responsibility of the PR assignee.

### CODE REVIEW

**Review by BogdanDenis — APPROVED** [review]

File: src/routes/ConsortiumManager/settings/authorization-policies/AuthorizationPoliciesSettings.test.js [hunk 1]
```diff
@@ -1,50 +1,69 @@
...
+  // beforeEach(() => {
+  //   useMemberSelectionContext
+  //     .mockClear()
+  //     .mockReturnValue({ activeMember: 'central' });
+  //   useAuthorizationRoles.mockClear().mockReturnValue({
+  //     roles: [],
+  //     isLoading: false,
+  //     onSubmitSearch: jest.fn(),
+  //   });
+  // });
```
Comment (Terala-Priyanka) [inline src/routes/ConsortiumManager/settings/authorization-policies/AuthorizationPoliciesSettings.test.js:45]: Do you need this commented code?

Comment (Terala-Priyanka) [inline src/routes/ConsortiumManager/settings/authorization-policies/AuthorizationPoliciesSettings.test.js:45]: Should you probably remove this commented code?

**Review by zburke — APPROVED** [review]
It's very difficult to tell what is happening in this PR because the diff is so big and the description doesn't mention why. The "related" PR you mention is, in fact, the crux of the work here: now that shared components are available in stripes-authz-components, you can leverage them here. Please say that :) 

As others have pointed out, please clean up the commented code in the test files, or include a comment about why you're preserving it and how/when you intend to un-comment it in future work.

File: src/routes/ConsortiumManager/settings/authorization-policies/AuthorizationPoliciesSettings.test.js [hunk 2]
```diff
@@ -1,50 +1,58 @@
...
-    renderWithRouter(<AuthorizationPoliciesSettings {...props} />);
-
-    const searchInput = screen.getByLabelText('ui-consortia-settings.authorizationPolicy.search');
+    expect(screen.getByText('PolicyFormContainer')).toBeInTheDocument();
+  });
 
-    expect(searchInput).toBeInTheDocument();
-    await userEvent.type(searchInput, searchQuery);
-    await userEvent.click(screen.getByRole('button', { name: 'ui-consortia-settings.authorizationPolicy.search' }));
+  it('should display edit authorization policy edit page', async () => {
```
[the remark this answers is not in the document]
Comment (alisher-epam) [reply src/routes/ConsortiumManager/settings/authorization-policies/AuthorizationPoliciesSettings.test.js]: updated

**Review by mariia-aloshyna — APPROVED** [review]

**Pull request merged**